text/0000-lts-release-strategy.md

```diff
@@ -0,0 +1,90 @@
+
+If a major release is not delivered for an extended period of time, the maintainers may designate a subsequent minor release as the next LTS release, for example if v3.1 is a LTS release and there is no need to increment to 4.0 for several quarters, the maintainers may decide to designate v3.4 as the next LTS release.
+
+## LTS 3rd digit patch releases
+
+For LTS releases, 3rd digit patch releases will be provided for bug and security fixes at least every three months, but may be more frequent if important fixes become available sooner. Significant new features and other changes will not be included in 3rd digit patch releases, in order to ensure the stability of the LTS release and reduce the risk of functional regressions and bugs.
+
+It is worth noting that the v1.4 LTS release did not follow the above principles. Had the v1.4 LTS release followed the above principles, the Raft ordering service feature would likely have been introduced in a v1.5 release, and Kafka to Raft migration in a v1.6 release. We mention only to illustrate the above principles.
+
+## LTS release duration
+
+Once a *subsequent* LTS release is designated, users can expect 3rd digit patch releases to address critical bugs on the *prior* LTS release for approximately 9 more months. The maintainers will determine which fixes should to be backported to the latest LTS release, versus which fixes need to be further backported to the prior LTS release. The overlap period is intended to provide users a time window to upgrade their nodes, and any channel capabilities (if required).
+
+## LTS release diagram
+The following diagram represents the LTS release cadence and duration using hypothetical Fabric releases:
+![LTS release diagram](../images/lts_release_diagram.png)
+
+## LTS and channel capabilities
+
+Channel capabilities will remain available for two LTS releases, but may be removed in subsequent LTS releases.  For example, v1.x capabilities such as legacy chaincode will be available through v2.2 (assuming v2.2 is a LTS release), but are expected to be removed before the subsequent LTS release. This provides one full LTS cycle for channels to transition to new capabilities. For example, a production Fabric network may perform a rolling upgrade from v1.4.x nodes to v2.2.x nodes, then enable v2.x channel capabilities to take advantage of the new chaincode lifecycle, and then eventually upgrade to v3.x LTS nodes, where the legacy chaincode lifecycle may not be available.

```

Comment:
This section has a couple issues for me.

First, I think using the word 'channel capabilities' is a bit overloaded.  Do you mean literally the things in the channel config we call capabilities? Until we having archive and pruning, we do not have the ability to remove any of them.

If this is meant more broadly to be 'features', and not 'capabilities', then this seems like a very firm line to take.  This could be misinterpreted to claim that an esoteric CLI flag from 2.2 must be supported through v3.1 even if usage is low and migration easy.

The idea here seems to be "there is a sane migration path from LTS to LTS", but this language is a bit strong to me.

Reply:
Changing to:
Features that are related to capabilities will remain available for two LTS releases, but may be removed in subsequent LTS releases.  For example, v1.x legacy chaincode management will be available through v2.2 (assuming v2.2 is a LTS release), but is expected to not be usable in the subsequent LTS release. This provides one full LTS cycle for channels to transition to new capabilities. For example, a production Fabric network may perform a rolling upgrade from v1.4.x nodes to v2.2.x nodes, then enable v2.x channel capabilities to take advantage of the new chaincode lifecycle, and then eventually upgrade to v3.x LTS nodes, where the legacy chaincode lifecycle may not be available.